lib/judge/controller.rb

```diff
@@ -9,11 +9,11 @@ def self.included(base)
     end
 
     def validation(params)
-      params = normalized_params(params)
-      if params_present?(params) && params_exposed?(params)
-        Validation.new(params)
+      norm_params = normalized_params(params)
```

Comment:
Yeah, I have a feeling that it's specifically because of the `dup`.

I don't totally understand it, but I see the results.

Check slack.. and let me know what you think.